cmd.go

```diff
@@ -116,13 +115,12 @@ func (b *Bot) executePassiveCommands(cmd *PassiveCmd) {
 		go func() {
 			defer wg.Done()
 
-			log.Println("Executing passive command: ", cmdName)
 			result, err := cmdFunc(cmd)
 			if err != nil {
 				log.Println(err)
 			} else {
 				mutex.Lock()
-				b.messageHandler(cmd.Channel, result, cmd.Nick)
+				handlers.Response(cmd.Channel, result, cmd.Nick)
 				mutex.Unlock()
```

Comment:
Guess we can but I can see the advange of a channel, it will have to block (as does the mutex) and we will have more code in this part, the cmd returns 2 values so we will hav ve something like `make(chan struct { msg string, err error})`.

If you'd like you can impement it and send a PR, I'm still not sure that channels will be better than this simple mutex.